purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -180,17 +182,30 @@ class Purchases @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE) intern
                             !this.finishTransactions,
                             null,
                             null,
-                            { info ->
+                            unsyncedSubscriberAttributesByKey,
+                            { info, _ ->
+                                subscriberAttributesManager.markAsSynced(
+                                    appUserID,
+                                    unsyncedSubscriberAttributesByKey,
+                                    emptyList()
```

Comment:
I was a bit confused at first about what `emptyList()` would be, it's kind of hard to guess from the method name. Maybe named params would make that clear

Reply:
otherwise defaulting to null might work so that you don't have to send the parameter in if it's not needed?

Reply:
I actually realized this should be forwarding the `attributeErrors` right? Idk why I sent `emptyList`